Issue with React 17

```
npm i react-sparkle
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: web@1.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from react-sparkle@1.0.8
npm ERR! node_modules/react-sparkle
npm ERR!   react-sparkle@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/package.json
+++ b/package.json
@@ -73,6 +73,6 @@
     ]
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": ">=16.8.0 <18"
   }
 }

